R/read_csv.R

```diff
@@ -196,13 +228,13 @@ read_sample_csv <- function(output_files) {
       id <- sampling_info$id
     } else {
       csv_file_info <- read_sample_info_csv(output_file)
-      # check if sampling info matches
       check <- check_sampling_csv_info_matches(sampling_info, csv_file_info)
       if (!is.null(check$error)) {
         stop(check$error)
       }
       not_matching <- c(not_matching, check$not_matching)
       sampling_info$id <- c(sampling_info$id, csv_file_info$id)
+      sampling_info$seed <- c(sampling_info$seed, csv_file_info$seed)
```

Comment:
I added this since the seed is different for each chain also

Reply:
Thanks!